docs/decisions/2022-11-tracing-with-otel.md

```diff
@@ -0,0 +1,66 @@
+# Tracing with Open Telemetry
+
+* **Status:** proposed
```

Comment:
We have the proposed status, but we don't have a strongly defined flow to go from proposed to approved.

Two paths forward:
* Minimal review that this DR is enough to have a good discussion, so merge now, discuss later, then update
* Use the PR to prompt a discussion, confirm the direction  and update to approved before merge.